src/wazuh_testing/utils/database.py

```diff
@@ -134,6 +134,25 @@ def get_sqlite_query_result(db_path: str, query: str) -> List[str]:
         services.control_service('start', daemon=WAZUH_DB_DAEMON)
 
 
+def get_sqlite_fetch_one_query_result(db_path: str, query: str) -> tuple:
```

Comment:
[db.execute_fetch_one_query](https://github.com/wazuh/qa-integration-framework/blob/1321b3bd96078745949ea2b55c3b5989f5e32387/src/wazuh_testing/tools/db_administrator.py#L41) is returning a list

```suggestion
def get_sqlite_fetch_one_query_result(db_path: str, query: str) -> list:
```

Reply:
You are right, but the [documentation](https://docs.python.org/3/library/sqlite3.html#sqlite3.Cursor.fetchone) mentions that a `tuple` is returned